src/data.json

```diff
@@ -219,15 +219,15 @@
         "atomic_number": "28",
         "media": "media/28nickel.gif",
         "name": "Nickel",
-        "phrase": "**Ni**ckel. In 1928, the year before the Great Depression, people had more money (a nickel) than in the next year ('29). The U.S. coin called a \"nickel\" used to be made of nickel.",
+        "phrase": "**Ni**ckel. In 1928, the year before the Great Depression, people had more money (a nickel) than in the next year ('29). The U.S. coin called a \"nickel\" is (partly) made of nickel.",
```

Comment:
Even as early at the 1866 Shield nickel—the first US 5-cent coin generally referred to as a "nickel"—the coin was 75% Cu and 25% Ni (as it is today).

(Technically, the coin has changed composition along the way—e.g., during WWII—but that needn't affect the memory-peg phrase.)